### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by htmldoug**
More readme.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -34,106 +62,135 @@ object Foo {
...
-So this is a shaded fork of uPickle. It is hard-shaded (instead of using
-sbt-assembly or something like that) because uPickle includes macros with
-hard-coded paths, so automatic shading isn't likely to work correctly.
+Case classes are serialized using the `apply` and `unapply` methods on their companion objects. This means that you can make your own classes serializable by giving them companions `apply` and `unapply`. `sealed` hierarchies are serialized as tagged unions: whatever the serialization of the actual object, together with the fully-qualified name of its class, so the correct class in the sealed hierarchy can be reconstituted later.
 
-## Differences
-The upstream https://github.com/lihaoyi/upickle macros serialize some things differently
-than other common libs like circe and play-json. Many of the differences have 
-well-reasoned motivations, but hinder adoption as a drop-in replacement.
+Anything else is not supported by default, but you can add support using Custom Picklers.
```
Comment (danny-goldberg-rally) [inline readme.md:91]: how would one get started on creating custom picklers?

File: misc.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Many of Rally Health's libraries need to work with JSON. However, we have
+found (through long painful experience) that letting them use Play-Json
+puts us into a particular form of Dependency Hell: the libraries wind up
+dependent on a *specific version* of Play-Json, which means that upgrading
+Play requires upgrading much of our ecosystem, which is a hassle. We want
+to decouple our libraries from Play as much as possible, to reduce this
+friction.
+
+So we are encouraging libraries to make use of weePickle instead: it's
```
Comment (JasnaMRB) [inline misc.md:11]: I _think_ you mean `uPickle` here. Then on line 19, to be clear, you can say, `So this, weePickle, is a shaded fork of uPickle`.

File: misc.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+popular, well-supported and fast.
```
Comment (JasnaMRB) [inline misc.md:12]: Maybe add a 1-2 sentence summary of what `uPickle (pronounced "micro-pickle") [that's for your readers, not you, Doug]` is here as opposed to `Play-Json` (other than "popular, well-supported and fast". Edit: can add your "Original Readme" notes here.

File: misc.md (hunk 2 above)
Comment (danny-goldberg-rally) [reply misc.md:11]: become an early contributor - put up a PR to make the changes that you want to see :)

**Pull request merged**